src/reducers/tokenPair.ts

```diff
@@ -1,18 +1,19 @@
 import { handleActions } from 'redux-actions'
 
-import { selectTokenAndCloseOverlay } from 'actions'
+import { selectTokenAndCloseOverlay, selectTokenPair } from 'actions'
 import { TokenPair } from 'types'
 import { TokenItemProps } from 'components/TokenItem'
 
-export default handleActions<TokenPair, TokenItemProps>(
+export default handleActions<TokenPair, TokenItemProps & TokenPair>(
```

Comment:
normally these .toString() are NOT required - I know you were running into issues with reducers but are you sure this is necessary? Normally just inputting the action export name is enough...

Reply:
It is either `.toString()` or `as any`. And `.toString()` is called either way, when casting object keys.

I don't mind changing if you feel it would be better

Reply:
in terms of micro op is `.toString()` faster?